src/features/unified-share-modal/ContactsField.js

```diff
@@ -55,20 +55,20 @@ class ContactsField extends React.Component<Props, State> {
         };
     }
 
-    addSuggestedContacts = (contacts: Array<Contact>) => {
+    getSuggestedCollaboratorArray = (): Array<SuggestedCollab> => {
         const { suggestedCollaborators = {} } = this.props;
+        return Object.keys(suggestedCollaborators).map(key => suggestedCollaborators[key]);
+    };
 
-        const suggestedSelectorOptions = contacts
-            .filter(option => {
-                const id = option.id;
-                return id && suggestedCollaborators[id.toString()];
-            })
-            .sort((optionA, optionB) => {
-                const currentSuggestedItemA = suggestedCollaborators[optionA.id.toString()];
-                const currentSuggestedItemB = suggestedCollaborators[optionB.id.toString()];
-                return currentSuggestedItemB.userScore - currentSuggestedItemA.userScore;
-            })
-            .slice(0, 3);
+    addSuggestedContacts = (contacts: Array<Contact>): Array<Contact> => {
+        const { suggestedCollaborators = {} } = this.props;
```

Comment:
Recommend using `static defaultProps` for default prop values to keep class behavior consistent.